aws/table_aws_wellarchitected_share_invitation.go

```diff
@@ -0,0 +1,175 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/wellarchitected"
+	"github.com/aws/aws-sdk-go-v2/service/wellarchitected/types"
+
+	wellarchitectedv1 "github.com/aws/aws-sdk-go/service/wellarchitected"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsWellArchitectedShareInvitation(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_wellarchitected_share_invitation",
+		Description: "AWS Well-Architected Share Invitation",
+		List: &plugin.ListConfig{
+			Hydrate: listWellArchitectedShareInvitations,
+			KeyColumns: []*plugin.KeyColumn{
+				{Name: "workload_name", Require: plugin.Optional},
+				{Name: "lens_name", Require: plugin.Optional},
+			},
+		},
+		GetMatrixItemFunc: SupportedRegionMatrix(wellarchitectedv1.EndpointsID),
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "lens_arn",
+				Description: "The ARN for the lens.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "lens_name",
+				Description: "The full name of the lens.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "permission_type",
+				Description: "Permission granted on a workload share.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "share_invitation_id",
+				Description: "The ID assigned to the share invitation.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "share_resource_type",
+				Description: "The resource type of the share invitation.",
```

Comment:
```suggestion
				Description: "The resource type of the shared invitation.",
```

Reply:
Which is the description from AWS? If it's the original, I think keeping it as `share` is OK since it's referring to a "share" resource.

Reply:
Yes, this one's the original from AWS